tests/AntDesign.TestKit/AntDesignTestBase.cs

```diff
@@ -22,6 +22,7 @@ public AntDesignTestBase(bool useMoq = false)
             //Needed for Tests using Overlay
             Services.AddScoped<AntDesign.JsInterop.DomEventService>(sp => new TestDomEventService(Context.JSInterop.JSRuntime, MockedDomEventListener));
             JSInterop.SetupVoid(JSInteropConstants.OverlayComponentHelper.DeleteOverlayFromContainer, _ => true);
+            JSInterop.Mode = JSRuntimeMode.Strict;
```

Comment:
Added this so even if a test changes it, all other tests still have it set to the default Strict. Not 100% sure the lifetime of this service but figured it was a good precaution.